README.md

```diff
@@ -33,41 +33,49 @@ Apache Traffic Control allows you to build a large scale content delivery networ
 
-## Documentation [^1]
+[![Go Unit Tests](https://github.com/apache/trafficcontrol/workflows/Go%20Unit%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/go.unit.tests.yaml?query=branch%3Amaster) [![Codecov](https://codecov.io/gh/apache/trafficcontrol/branch/master/graph/badge.svg)](https://app.codecov.io/github/apache/trafficcontrol/tree/master)
+
+[![TO Go Client Integration Tests](https://github.com/apache/trafficcontrol/workflows/TO%20Go%20Client%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/traffic-ops.yml?query=branch%3Amaster) [![Codecov](https://codecov.io/gh/apache/trafficcontrol/branch/master/graph/badge.svg?flag=traffic_ops_integration)](https://app.codecov.io/github/apache/trafficcontrol/tree/master/traffic_ops)
+
+[![Traffic Router Tests](https://github.com/apache/trafficcontrol/workflows/Traffic%20Router%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tr.tests.yaml?query=branch%3Amaster) [![Codecov](https://codecov.io/gh/apache/trafficcontrol/branch/master/graph/badge.svg?flag=traffic_router_unit)](https://app.codecov.io/github/apache/trafficcontrol/tree/master/traffic_router)
+
+[![TP Integration Tests](https://github.com/apache/trafficcontrol/workflows/TP%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tp.integration.tests.yml?query=branch%3Amaster) 
+[![TM Integration Tests](https://github.com/apache/trafficcontrol/workflows/TM%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tm.integration.tests.yml?query=branch%3Amaster)
+[![TR Ultimate Test Harness](https://github.com/apache/trafficcontrol/workflows/TR%20Ultimate%20Test%20Harness/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tr-ultimate-test-harness.yml?query=branch%3Amaster)
+[![T3C Integration Tests](https://github.com/apache/trafficcontrol/workflows/T3C%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/cache-config-tests.yml?query=branch%3Amaster)
+[![TC Health Client Integration Tests](https://github.com/apache/trafficcontrol/workflows/TC%20Health%20Client%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/health-client-tests.yml?query=branch%3Amaster)
+
+
+---
+
+## Documentation [^2]
 * [Intro](http://traffic-control-cdn.readthedocs.io/en/latest/index.html)
 * [CDN Basics](http://traffic-control-cdn.readthedocs.io/en/latest/basics/index.html)
 * [Traffic Control Overview](http://traffic-control-cdn.readthedocs.io/en/latest/overview/index.html)
 * [Administrator's Guide](http://traffic-control-cdn.readthedocs.io/en/latest/admin/index.html)
 * [Developer's Guide](http://traffic-control-cdn.readthedocs.io/en/latest/development/index.html)
 * [Traffic Ops API](https://traffic-control-cdn.readthedocs.io/en/latest/api/index.html)
 
-## Components [^1]
+## Components [^2]
```

Comment:
FYI, link references can be named so that you don't need to renumber them when making changes like this
e.g.

```markdown
Claim [^citation]

...

[^citation]: https://some-source.invalid/paper/by/a/scientist/or/something
```